webpack/server/webpack.config.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path')
+const nodeExternals = require('webpack-node-externals')
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin')
+
+const ROOT = path.join(__dirname, '/../../')
+
+const config = {
+  mode: process.env.NODE_ENV,
+  target: 'node',
+  node: {
+    __dirname: true,
+    __filename: true,
+  },
+  externals: [nodeExternals()],
+  entry: path.join(ROOT, 'lib/app.ts'),
+  output: {
+    path: path.join(ROOT, 'dist/server'),
```

Comment:
[Just wondering] Do you plan to compile client-side assets to dist/client? I wondered about why using more deep hierarchy.

Reply:
I don't have such a plan at the moment.
However, I just meant to show the domain explicitly.